cli/configure.js

```diff
@@ -1,78 +1,123 @@
-                thumbnail  : results.defaultAssetThumbnail,
-            },
-            auth: {
-                sessionKey: results.sessionKey,
-            },
-            details: {
-                port       : results.port,
-                title      : results.title,
-                host       : results.host,
-                description: results.description,
-                twitter    : results.twitter,
-            },
-            publishing: {
-                primaryClaimAddress     : results.primaryClaimAddress,
-                uploadDirectory         : results.uploadDirectory,
-                thumbnailChannel        : results.thumbnailChannel,
-                thumbnailChannelId      : results.thumbnailChannelId,
-                additionalClaimAddresses: [],
-                disabled                : false,
-                disabledMessage         : 'Please check back soon',
-            },
-        };
-        const fileLocation = Path.resolve(__dirname, '../config/siteConfig.json');
-        const fileContents = JSON.stringify(siteResponses, null, 2);
-        fs.writeFileSync(fileLocation, fileContents, 'utf-8');
-    })
-    .then(() => {
-        return inquirer.prompt(slackQuestions);
-    })
-    .then(results => {
-        const fileLocation = Path.resolve(__dirname, '../config/slackConfig.json');
-        const fileContents = JSON.stringify(results, null, 2);
-        fs.writeFileSync(fileLocation, fileContents, 'utf-8');
-    })
-    .then(() => {
-        console.log('All done!');
-        process.exit(0);
-    })
-    .catch(error => {
-        console.log(error);
-        process.exit(1);
-    });
+  .prompt(loggerQuestions)
+  .then(results => {
+      const fileLocation = Path.resolve(__dirname, '../config/loggerConfig.json');
+      const fileContents = JSON.stringify(results, null, 2);
+      fs.writeFileSync(fileLocation, fileContents, 'utf-8');
+  })
+  .then(() => {
+      return inquirer.prompt(mysqlQuestions);
+  })
+  .then(results => {
+      const fileLocation = Path.resolve(__dirname, '../config/mysqlConfig.json');
+      const fileContents = JSON.stringify(results, null, 2);
+      fs.writeFileSync(fileLocation, fileContents, 'utf-8');
+  })
+  .then(() => {
+    // check for lbrynet connection & retrieve a default address
+    console.log('\nRetrieving primary claim address...');
+    return axios
+      .post('http://localhost:5279', {
+        method: 'wallet_list',
+      })
+      .then(response => {
+        if (response.data) {
+          primaryClaimAddress = response.data.result[0];
+          console.log('Primary claim address:', primaryClaimAddress);
+          return;
+        }
+        throw new Error('No data received from lbrynet');
+      }).catch(error => {
+        throw new Error(`Error received from lbrynet.  Please start lbry and try again. Error: ${error}`);
+      })
+  })
+  .then(() => {
+    // create thumbnail address
+    console.log('\nCreating thumbnail channel...');
```

Comment:
It might be helpful to describe what this means. "Creating a channel to publish your thumbnails to" or something more descriptive

Reply:
good idea.  updated.